script/app.js

```diff
@@ -0,0 +1,69 @@
+let addToDoButton = document.querySelector(".task__button-add");
+let toDoContainer = document.querySelector(".todo__container");
+let todoInput = document.querySelector(".task__input");
+let createTask = document.querySelector(".task-board__button--create");
+let taskCardNew = document.querySelector(".task__card-new");
+let taskContainer = document.querySelector(".task__container");
+let taskCard = document.querySelector(".task__card");
+
+createTask.addEventListener("click", function () {
+    taskCard.style.display =
+        taskCard.style.display === "none" || !taskCard.style.display
+            ? "block"
+            : "none";
+    createTask.innerText =
+        taskCard.style.display === "none" ? "+Create Task" : "Hide Task";
+});
+
+const todos = [];
+
+function generateUniqueId() {
+    return new Date().getTime();
+}
+function sanitizeInput(input) {
```

Comment:
It's better to keep this common function in different shared file

Reply:
done